src/autoqasm/types/types.py

```diff
@@ -152,10 +152,22 @@ def __init__(self, *args, annotations: str | Iterable[str] | None = None, **kwar
         )
         self.name = program.get_program_conversion_context().next_var_name(oqpy.FloatVar)
 
+    def __floordiv__(self, other):
+        raise NotImplementedError("Integer division is supported by OpenQASM.")
```

Comment:
@jcjaskula-aws thank you very much for the feedback! I have a follow-up question if you don't mind. In the PR, the floor division is not "directly" implemented, but rather intercepted by the transpiler and replaced by other operations. I originally added these class methods to trick linters into accepting the integer division involving `IntVar`s/`FloatVar`s, and figured it would make sense to raise an Exception if a user tried to actually call the methods.

Would you suggest I raise a different type of exception, maybe one related to AutoQASM not supporting integer division, or rather remove these methods altogether?

Reply:
Super clear! I just committed the changes.